<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>451</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="adjustments">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <attribute name="title">
      <string>Adjustments:</string>
     </attribute>
     <widget class="QDoubleSpinBox" name="particle_value">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>182</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
     <widget class="QSlider" name="width_slid">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>33</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="method_lbl">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Choose the method:</string>
      </property>
     </widget>
     <widget class="QLabel" name="width_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>33</y>
        <width>43</width>
        <height>16</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Width:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="density_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>156</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Particle Density</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>172</x>
        <y>119</y>
        <width>16</width>
        <height>101</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="method_list">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>142</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string notr="true">random</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>center</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="height_value_label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>63</y>
        <width>26</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>value</string>
      </property>
     </widget>
     <widget class="QLabel" name="height_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>64</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
     <widget class="QLabel" name="a_lbl">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>19</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>a=</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="quantity_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>116</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Particle Quantity</string>
      </property>
     </widget>
     <widget class="QLabel" name="width_value_label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>26</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>value</string>
      </property>
     </widget>
     <widget class="QSlider" name="height_slid">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>64</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="a_value">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="modify_model">
     <attribute name="title">
      <string>Modify the model:</string>
     </attribute>
     <widget class="QPushButton" name="save_modified">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>SAVE</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSlider" name="radius_slider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="radius_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Radius co≈õtam opis opis bla bla bla</string>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="text">
       <string>Radius:</string>
      </property>
     </widget>
     <widget class="QLabel" name="alpha_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Alpha bla bla blab :3</string>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="text">
       <string>Alpha =</string>
      </property>
     </widget>
     <widget class="QLabel" name="beta_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Beta cium cium cium &lt;3</string>
      </property>
      <property name="text">
       <string>Beta =</string>
      </property>
     </widget>
     <widget class="QLabel" name="v_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Jestem szybki...</string>
      </property>
      <property name="text">
       <string>Velocity =</string>
      </property>
     </widget>
     <widget class="QLabel" name="radius_value_label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>26</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>value</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="alpha_value">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <double>359.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="beta_value">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <double>359.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="velocity_value">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <double>5.000000000000000</double>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="visualization_group">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>571</width>
      <height>651</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Visualization:</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>551</width>
       <height>591</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="okno_grid_2" rowstretch="0,0,0" columnstretch="0,0">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="1" column="1">
       <widget class="QSlider" name="timestep_slid_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="timestep_lbl_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Time step:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QWidget" name="scatter_box_2" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(107, 211, 204);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="value_timestep_lbl">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>value</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="open_btn">
       <property name="text">
        <string>Open Data File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_btn">
       <property name="text">
        <string>Save Data to...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>451</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="3" column="0">
      <widget class="QWidget" name="terminal_box" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="run_btn">
       <property name="text">
        <string>RUN</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QProgressBar" name="progress_bar">
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
